docs/ui/insight/tools.md

```diff
@@ -0,0 +1,109 @@
+* Discover your databases in Redis Enterprise Cluster and databases with Flexible plans in Redis Cloud.
+* Use a form to enter your connection details and add any Redis database running anywhere (including OSS Cluster, Sentinel).
+
+<img src="../images/Databases.png">
+
+## Browser
+
+Browser offers [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) support for lists, hashes, strings, sets, sorted sets, streams, as well as for [RedisJSON](/docs/stack/json).
+
+* Browse, filter, and visualize your key-value Redis data structures. 
+
+* Group keys according to their namespaces.
+
+  <img src="../images/Browser.png">
+
+* View, validate, and manage your key values in a human-readable format using formatters that prettify and highlight data in different formats (for example, Unicode, JSON, MessagePack, HEX, and ASCII) in the Browser tool.
+
+  <img src="../images/data_formatting.png">
+
+## Profiler
+
+Analyze every command sent to Redis in real time.
+
+<img src="../images/Profiler.png">
+
+## CLI
+
+The CLI is accessible at any time within the application. It:
+
+* Employs type-ahead help to deliver intuitive assistance.
+* Includes a command helper that lets you filter and search for Redis commands.
+
+Try executing Redis commands as shown here:
+
+
+<img src="../images/CLI.png">
+
+## Workbench
+
+Workbench is a command-line interface with intelligent command auto-complete and complex data visualizations that lets you run commands against your Redis server. 
+Find the workbench on the left sidebar. It displays click-through examples of Redis capabilities. You can also see a number of metrics always on display within the database workspace. These metrics get updated every 5 seconds. The metrics include CPU, number of keys, commands/sec, network input, network output, total memory, and number of connected clients. Workbench provides: 
+
+* Built-in examples that help you discover Redis and Redis Stack capabilities using the built-in guides.
+* Command auto-complete support for all capabilities in Redis and Redis Stack.
+* Visualizations of your [RediSearch](https://oss.redis.com/redisearch/) index, queries, and aggregations.
+* Visualizations of your [RedisGraph](https://oss.redis.com/redisgraph/).
+
+  <img src="../images/Workbench_Graph.png">
+
+* Visualizations of your [RedisTimeSeries](https://oss.redis.com/redistimeseries/) data.
+
+  <img src="../images/Workbench_TimeSeries.png">
+
+## Database analysis
+
+Use the database analysis tool to optimize the performance and memory usage of your Redis database. Check data type distribution and memory allocation and review the summary of key expiration time and memory to be freed over time. Inspect the top keys and namespaces sorted by consumed memory or key length and count of keys, respectively. Capture and track the changes in your database by viewing historical analysis reports. Next figure shows a sample database analysis report.
```

Comment:
Please add information about database recommendations, for example:

Use recommendations based on developer best practices to:
- reduce memory usage
- improve application performance
- increase the security of your database

Recommendations are generated specifically for your Redis or Redis Stack database.
<img width="1268" alt="image" src="https://user-images.githubusercontent.com/99594890/221000467-34c3cb1e-caeb-4696-bafa-61ba194dc747.png">

Reply:
Updated. Will add the graphic in the next iteration.